source/backend/api/Services/TakeService.cs

```diff
@@ -75,7 +75,7 @@ public IEnumerable<PimsTake> UpdateAcquisitionPropertyTakes(long acquisitionFile
             var currentAcquisitionStatus = Enum.Parse<AcquisitionStatusTypes>(currentAcquistionFile.AcquisitionFileStatusTypeCode);
 
             // No user can update the takes when the File is not Active/Draft
-            if (!_statusSolver.CanEditTakes(currentAcquisitionStatus))
+            if (!_statusSolver.CanEditTakes(currentAcquisitionStatus) && !_user.HasPermission(Permissions.SystemAdmin))
```

Comment:
actually, this doesn't seem correct. I think even admins can't edit the takes when the file is complete.

Reply:
Ah good to know. When I merged eduardo's changes I thougt the story was about allowing sys-admins to delete takes. There is a story about it in this sprint given there are rules around disposed properties and takes